packages/cli/test/helpers/stubs/post.ts

```diff
@@ -0,0 +1,26 @@
+import * as nock from 'nock'
+
+export function collaborators() {
+  return nock('https://api.heroku.com:443')
+    .post('/apps/myapp/collaborators', {
+      user: 'raulb@heroku.com',
+    }).reply(200)
+}
+
+export function teamAppCollaborators(email = 'raulb@heroku.com', permissions: string[] = [], response: {code?: number, description?: Record<string, unknown>} = {}) {
```

Comment:
The `response` here doesn't match the signature for this API. A `POST` to `/teams/apps/${appIdentity}/collaborators` returns a `TeamAppCollaborator` which does not contain a `description` property.

Reply:
Looking at this more closely, it appears as though `description?: Record<string, unknown>}` is a placeholder and is used in place of the concrete `TeamAppCollaborator` type. Would it make sense to update this?

Reply:
Nothing! I broke nothing.

Reply:
I take it back. I was mistaken and I had fixed the wrong file the first time. When I removed the params from the new post.ts file, it did break things. Apparently we are using that param so that we can test an error message.